tests/mysql/types.rs

```diff
@@ -196,6 +199,17 @@ test_type!(decimal<sqlx::types::BigDecimal>(
     "CAST(12345.6789 AS DECIMAL(9, 4))" == "12345.6789".parse::<sqlx::types::BigDecimal>().unwrap(),
 ));
 
+#[cfg(feature = "decimal")]
+test_type!(decimal<sqlx::types::Decimal>(MySql,
+    "CAST(0 as DECIMAL(0, 0))" == sqlx::types::Decimal::from_str("0").unwrap(),
+    "CAST(1 AS DECIMAL(1, 0))" == sqlx::types::Decimal::from_str("1").unwrap(),
+    "CAST(10000 AS DECIMAL(5, 0))" == sqlx::types::Decimal::from_str("10000").unwrap(),
+    "CAST(0.1 AS DECIMAL(2, 1))" == sqlx::types::Decimal::from_str("0.1").unwrap(),
+    "CAST(0.01234 AS DECIMAL(6, 5))" == sqlx::types::Decimal::from_str("0.01234").unwrap(),
+    "CAST(12.34 AS DECIMAL(4, 2))" == sqlx::types::Decimal::from_str("12.34").unwrap(),
+    "CAST(12345.6789 AS DECIMAL(9, 4))" == sqlx::types::Decimal::from_str("12345.6789").unwrap(),
```

Comment:
We had an issue with numbers for `BigDecimal -> PgNumeric` that had a digit count that was divisible by 4, e.g. `1234`, `1234.5678`, `12345678`. Can you please add these to the tests to make sure we don't regress since the math is very similar?

Reply:
Well, you copied the unit tests from `BigDecimal` which include tests for that regression so I guess that's not technically necessary here.